tests/introspection/ApiCtorInitTest.cs

```diff
@@ -477,6 +477,11 @@ protected virtual bool Match (ConstructorInfo ctor, Type type)
 			case "UICollectionViewCompositionalLayout":
 				// Explicitly disabled ctors - (instancetype)init NS_UNAVAILABLE;
 				return true;
+			case "NSPickerTouchBarItem":
+				if (ctor.ToString () == $"Void .ctor(System.String)")
```

Comment:
please add a comment why this ctor is being disabled (like the above cases already have)

there's nothing in `/Applications/Xcode112-beta2.app//Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/AppKit.framework/Versions/C/Headers/NSPickerTouchBarItem.h` to suggest it should not be available.